notebooks/official/workbench/spark/spark_ml.ipynb

```diff
@@ -0,0 +1,1623 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    def manhattan_dist(lat1, lon1, lat2, lon2):
</pre><p>I would make this a little clearer by changing <code>lat1</code> to be <code>lat_start</code> etc.</p><p> </p><p>Also, double check the comment here, as I think there may be a mismatch in the description.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/838/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='55'/>